src/main/java/it/gov/pagopa/cgn/portal/email/EmailNotificationService.java

```diff
@@ -56,9 +60,14 @@ public void sendSyncMessage(EmailParams emailParams, String trackingKey) throws
             helper.setFrom(emailParams.getMailFrom());
             helper.setTo(emailParams.getMailToList().toArray(new String[0]));
 
+            if (Objects.nonNull(emailParams.getMailCCList()) && !emailParams.getMailCCList().isEmpty()) {
```

Comment:
Using the optional as commented in EmailParams qould have simplified this code to something similar to line 67.